### PR TITLE
Revise the TCS requirement

### PR DESCRIPTION
**Pull request by nurupo**
Based on the discussion in https://github.com/Tox/tox.chat/issues/224

### CODE REVIEW

**Comment by nurupo** [comment]
@JFreegman @sudden6 @Zer0-One does this sound good?

File: README.md [hunk 1]
```diff
@@ -56,8 +56,13 @@ The website doesn't have multilingual support at the moment, but it's planned to
...
-Since recently, we require clients to comply with [Tox Client Standard (TCS)](https://tox.gitbooks.io/tox-client-standard/content/index.html) to be added to our website. We don't enforce that for the clients that were added to the website before the requirement, but we expect them to get TCS compliant over time.
+Open an Issue/PR for the client addition.
 
-Once your client is TCS compliant, you can open an Issue/PR for the client addition.
+It used to be required that clients comply with the [Tox Client Standard (TCS)](https://tox.gitbooks.io/tox-client-standard/content/index.html) in order to be added to the website.
+While [it was decided not to require that anymore](https://github.com/Tox/tox.chat/issues/224), we still ask that you read through the TCS and try to comply at least with the security related points.
 
-Note that a client doesn't have to be TCS compliant to be added to our [Wiki](https://wiki.tox.chat/clients).
+When submitting a request for client addition, keep in mind that we expect the clients we list to be of high quality.
+It's not our intention to list every single Tox client in the existence.
```
Comment (sudden6) [inline README.md:65]: I think it's `in existence`

**Comment by sudden6** [comment]
Since TCS is not updated and it's practically impossible to fix it's issues I agree with not requiring it.

**Comment by nurupo** [comment]
@JFreegman and @Zer0-One lgtm'ed on irc.

**Pull request merged**